etc/links.profile

```diff
@@ -0,0 +1,59 @@
+# Firejail profile for links
+# Description: Text WWW browser
+# This file is overwritten after every install/update
+#
+# Persistent local customizations
+include links.local
+# Persistent global definitions
+# include globals.local
```

Comment:
Why this is commented?

Reply:
I saw somewhere `let include globals.local commented`. But that does not apply here, correct?

Reply:
To prevent a double include of globals.local, `include globals.local` is commented in profile which include a other profile which include globals.local. (see redirect_alias-profile.template).